addons/payment/views/payment_templates.xml

```diff
@@ -244,6 +265,11 @@
                             <span class="payment_option_name" t-esc="token.name"/>
                             <!-- === "V" check mark === -->
                             <t t-call="payment.verified_token_checkmark"/>
+                            <!-- === "Unpublished" badge === -->
+                            <span t-if="not token.env.user.user_has_groups('base.group_portal') and token.acquirer_id.published == False"
```

Comment:
Yes, since to portal users it wouldn't make much sense to see "Unpublished" next to their token, would it? I agree, it would be weird to see their token here, but not on checkout, but we don't want them to pay through that token anymore, so it should stay removed from checkout. 
The reason it's still on the manage form is that on the other hand it would be weird eg. to be paying a subscription through a token they don't have access to anymore.
I can't think of anything though to show instead of "Unpublished". "Expiring" would be the translation of the german word I have in mind, but it can easily be understood wrong.

Reply:
Let's keep it like this then! We'll reconsider if someone complains, but I don't think anyone will ever do. :smile: